content/threat-modelling.md

```diff
@@ -4,28 +4,28 @@ date: "2020-10-01"
+The below guidance is an enhancement of the four step OWASP threat modeling recommended for cloud native capabilities.
 
 ### End-to-end architecture
 A clear understanding of the organization's or individual's cloud native architecture should result in data impact guidance and classifications. This helps teams organize data distribution within the architecture as well as the additional protection mechanisms for it later on. cloud native diagrams and documentation should not only include the core components of the overall system design but should also take into consideration the location of the source code, the buckets and other storage mechanisms in use, and any additional aspects of the software development cycle. These are all areas that must be considered when initiating threat modeling for cloud native.
 
 ### Threat Identification
 When considering threats specific to an organization's cloud native capabilities, it is recommended to leverage a mature, well-used model of threats such as STRIDE or OCTAVE. Common threats organizations may wish to consider for their cloud native architectures includes, but is not limited to:
 
-Spoofing a cluster admin by stealing the authentication credentials via a social engineering attack
-Tampering of an API server config file or certificate could result in failed API server restart or mutual TLS authentication failures
-Repudiation of actions of an attacker because of disabled or misconfigured API auditing could result in a lack of evidence of a potential attack
-
+- Spoofing a cluster admin by stealing the authentication credentials via a social engineering attack
+- Tampering of an API server config file or certificate could result in failed API server restart or mutual TLS authentication failures
+- Repudiation of actions of an attacker because of disabled or misconfigured API auditing could result in a lack of evidence of a potential attack
 - Information disclosure is possible if an attacker compromises a running workload and is able to exfiltrate data to an external entity
 - Denial of Service (DoS) resulting from a pod that does not have resource limits applied therefore consumes the entire node level CPU and memory, worker node is then lost
 - Elevation of privilege could happen if a pod is running with unrestricted or higher privileged pod security policy or by modifying the security context of a pod or a container
 
 Threat actors to consider for cloud native security are consistent with existing threat modeling:
 
-- Malicious insider
-- Uninformed insider
-- Malicious outsider
-- Uninformed outsider
+- Malicious insider - An actor with malicious intent and with authorization to perform actions within the modelled system.
+- Uninformed insider - An actor that with authorization to perform actions within the modelled system (assume anyone can be duped).
+- Malicious outsider - An actor outside of the system. They could be attacking via the internet, via supply chain, via physical perimeter etc.
```

Comment:
uninformed outsider was removed.   any reason?  it is possible users outside of the system may inadvertently introduce issues.  For example, high school students writing scripts that create an unclosed loop which performs a DoS against their college or perhaps a banking website.

Reply:
I see your point, these actors exist. I'm not sure how they're useful in a threat model though. An damaging action available (like the DoS you describe above) to an Uninformed Outside is also available to someone with malicious intent. Likewise the mechanisms for constraining "accidental bad behavior" are likely to be the same as those for a Malicious actor.

Reply:
it may beneficial to add a statement to the effect of: "for the purposes of threat modeling, attack avenues of an uninformed outsider are represented in a malicious outsider without intent" for both this and the CNSWP.

Reply:
Done, at least a variation of that in my own words (line 30)